.github/CODEOWNERS

```diff
@@ -215,6 +215,13 @@ sdk/confidentialledger/azure-confidentialledger                      @lynshi
 /sdk/ml/azure-ai-ml/azure/ai/ml/_schema/workspace/connections                 @paulshealy1 @singankit @diondrapeck @luigiw @harneetvirk @kdestin @MilesHolland @needuv @nthandeMS
 /sdk/ml/azure-ai-ml/azure/ai/ml/entities/_workspace/connections               @paulshealy1 @singankit @diondrapeck @luigiw @harneetvirk @kdestin @MilesHolland @needuv @nthandeMS
 
+# PRLabel: %ML-ImportJob
```

Comment:
Please add a new rule for this label and associated paths to [fabricbot.json](https://github.com/Azure/azure-sdk-for-python/blob/main/.github/fabricbot.json#L1064)